Sources/DatadogSDKTesting/DDTracer.swift

```diff
@@ -57,7 +58,7 @@ internal class DDTracer {
         let version = (bundle.infoDictionary?["CFBundleShortVersionString"] as? String) ?? "unknown"
 
         var endpoint: Endpoint
-        switch env.ddEndpoint {
+        switch conf.endpoint {
             case "us", "US", "us1", "US1", "https://app.datadoghq.com", "app.datadoghq.com", "datadoghq.com":
                 endpoint = Endpoint.us1
             case "us3", "US3", "https://us3.datadoghq.com", "us3.datadoghq.com":
```

Comment:
it will be nice to have these as constants somewhere else....

Reply:
yeah. But for now I'm trying to refactor one part at a time. `DDTracer` is in a pipeline for refactoring